sys/net/gnrc/netif/gnrc_netif.c

```diff
@@ -859,25 +881,47 @@ static inline bool _addr_anycast(const gnrc_netif_t *netif, unsigned idx)
     return (netif->ipv6.addrs_flags[idx] & GNRC_NETIF_IPV6_ADDRS_FLAGS_ANYCAST);
 }
 
-static int _addr_idx(const gnrc_netif_t *netif, const ipv6_addr_t *addr)
+static int _idx(const gnrc_netif_t *netif, const ipv6_addr_t *addr, bool mcast)
 {
     if (!ipv6_addr_is_unspecified(addr)) {
+        const ipv6_addr_t *iplist = (mcast) ? netif->ipv6.groups : netif->ipv6.addrs;
         for (unsigned i = 0; i < GNRC_NETIF_IPV6_ADDRS_NUMOF; i++) {
```

Comment:
See https://github.com/RIOT-OS/RIOT/pull/9683#discussion_r207231138

Reply:
(since you changed stuff on your rebase GitHub thinks it was "fixed", but it isn't)

Reply:
yeah, but now it should be